### PR TITLE
behat tests

### PR DESCRIPTION
**Pull request by elenabc**

### CODE REVIEW

File: dev/tests/behavioral/features/front/product-purchase.feature [hunk 1]
```diff
@@ -0,0 +1,62 @@
+@javascript
```
Comment (dsmolovich) [inline dev/tests/behavioral/features/front/product-purchase.feature]: Nice change

File: dev/tests/behavioral/features/front/product-purchase.feature [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      | street[0]          | 5 Main street   |
+      | city               | King            |
+
+    And I select "Florida" from "region_id"
+    And I fill in the following:
+      | postcode           | 12345           |
+    And I select "United States" from "country_id"
+    And I fill in the following:
+      | telephone          | 4166666666      |
+    And I click on the element with xpath "//*[@id='checkout-shipping-method-load']/table/tbody/tr[1]/td[1]/input"
```
Comment (dsmolovich) [inline dev/tests/behavioral/features/front/product-purchase.feature]: Nice change

Comment (dsmolovich) [inline dev/tests/behavioral/features/front/product-purchase.feature]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

File: dev/tests/behavioral/features/front/product-purchase.feature (hunk 1 above)
Comment (dsmolovich) [inline dev/tests/behavioral/features/front/product-purchase.feature]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

File: dev/tests/behavioral/behat.yml [hunk 3]
```diff
@@ -15,3 +15,4 @@ default:
                      "version": ""
```
Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

File: dev/tests/behavioral/behat.yml [hunk 4]
```diff
@@ -15,3 +15,4 @@ default:
...
     filters:
```
Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

File: dev/tests/behavioral/behat.yml [hunk 5]
```diff
@@ -15,3 +15,4 @@ default:
...
         tags: "~@skip"
+
```
Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

File: dev/tests/behavioral/behat.yml (hunk 3 above)
Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

File: dev/tests/behavioral/behat.yml (hunk 4 above)
Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

Comment (dsmolovich) [inline dev/tests/behavioral/behat.yml]: Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

**Comment by dsmolovich** [comment]
test

**Comment by dsmolovich** [comment]
This is a test comment

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed. Failed suites:
- Behat tests [1]

Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests [failed: **1** feature(s)]

Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this PR@9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
I'm happy to let you know that this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 successfully passed all tests. :o)

Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
I'm happy to let you know that this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 successfully passed all tests.
Great job!!! :o)

Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
I'm happy to let you know that this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 successfully passed all tests.
Great job!!!
:o)
Please see [Test Run Log](http://localhost/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
I'm happy to let you know that this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 successfully passed all tests.
Great job!!!
:o)
Please see [Test Run Log](https://taasty.net/test-run/317/) for more details.

**Comment by dsmolovich** [comment]
I'm happy to let you know that this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 successfully passed all tests.
Great job!!!
:o)
Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
I'm happy to let you know that this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 successfully passed all tests.
Great job!!!
:o)
Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 09ca68f9a0b65a5a1cb23720d423f19f80d1587f failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Behat tests - **3** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/333/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Behat tests - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/317/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 84c3a5a6060555e3a4662901afc20f7348205be0 failed to successfully pass all tests.
**16 feature(s) failed**. Failed suites:
- Behat tests - **4** feature(s) failed
- Behat tests - **4** feature(s) failed
- Behat tests - **4** feature(s) failed
- Behat tests - **4** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/327/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 3c1d16809c66fcf7a6d6144cdfe8cf7b57330f7c failed to successfully pass all tests.
**4 feature(s) failed**. Failed suites:
- Behat tests - **4** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/329/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ ce4aaa16f50a9433d9c7ab7001f464bfa7ae8514 failed to successfully pass all tests.
**4 feature(s) failed**. Failed suites:
- Behat tests - **4** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/330/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 02d501462530ff6da7c157e737a6198516c52c1e failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Behat tests - **3** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/331/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ eb0697b3d02548d1b6e87b4c2d02159b556b0d43 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Behat tests - **3** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/332/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 09ca68f9a0b65a5a1cb23720d423f19f80d1587f failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Behat tests - **3** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/333/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 09ca68f9a0b65a5a1cb23720d423f19f80d1587f failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Behat tests - **3** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/333/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 09ca68f9a0b65a5a1cb23720d423f19f80d1587f failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Behat tests - **3** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/333/) for more details.

**Comment by dsmolovich** [comment]
Sorry, but unfortunately this Pull Request @ 09ca68f9a0b65a5a1cb23720d423f19f80d1587f failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Behat tests - **3** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/333/) for more details.

**Comment by dsmolovich** [comment]
test

**Comment by dsmolovich** [comment]
test

**Comment by dsmolovich** [comment]
Test Run 868:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/868/) for more details.

**Comment by dsmolovich** [comment]
Test Run 868:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/868/) for more details.

**Comment by dsmolovich** [comment]
Test Run 868:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/868/) for more details.

**Comment by dsmolovich** [comment]
Test Run 868:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/868/) for more details.

**Comment by dsmolovich** [comment]
Test Run 868:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/868/) for more details.

**Comment by dsmolovich** [comment]
Test Run 868:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/868/) for more details.

**Comment by dsmolovich** [comment]
Test Run 868:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/868/) for more details.

**Comment by dsmolovich** [comment]
I'm happy to let you know that this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 successfully passed all tests.
Please check [Test Run Log](https://taasty.net/test-run/view/2174/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2178:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Customer UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2178/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2179:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Customer UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2179/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2179:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Customer UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2179/) for more details.

**Comment by taasty** [comment]
Test Run 2179:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Customer UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2179/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2179:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Customer UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2179/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2180:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed
- Admin UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2180/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2183:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2183/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2199:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed
- Admin UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2199/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2209:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2209/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2210:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2210/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2211:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2211/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2213:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2213/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2215:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**1 feature(s) failed**. Failed suites:
- Customer UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2215/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2218:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed
- Admin UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2218/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2220:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed
- Admin UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2220/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2222:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**2 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2222/) for more details.

**Comment by dsmolovich** [comment]
Test Run 2224:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed
- Admin UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2224/) for more details.

**Comment by taasty** [comment]
Test Run 2224:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed
- Admin UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2224/) for more details.

**Comment by taasty** [comment]
Test Run 2224:
Sorry, but unfortunately this branch @ 101f541b86f6f92f931a545a6c8c5c1c37580e61 failed to successfully pass all tests.
**3 feature(s) failed**. Failed suites:
- Customer UI - **2** feature(s) failed
- Admin UI - **1** feature(s) failed

Please see [Test Run Log](https://taasty.net/test-run/view/2224/) for more details.

**Pull request closed without merging**